src/Generating/VillageGen.cpp

```diff
@@ -336,14 +336,15 @@ cVillageGen::cVillageGen(
 	const AStringVector & a_PrefabsToLoad
 ) :
 	super(a_Seed, a_GridSize, a_GridSize, a_MaxOffset, a_MaxOffset, a_MaxSize, a_MaxSize, 100),
-	m_Noise(a_Seed + 1000),
 	m_MaxDepth(a_MaxDepth),
 	m_MaxSize(a_MaxSize),
 	m_MinDensity(a_MinDensity),
 	m_MaxDensity(a_MaxDensity),
 	m_BiomeGen(a_BiomeGen),
 	m_HeightGen(a_HeightGen)
 {
+	m_Noise = a_Seed + 1000;
```

Comment:
This is a problem. This changes the generated structure, in the old code there are two instances of `cNoise`, each with a separate seed. In the new code, only one instance with this seed is used.

Instead, you should rename this class's `m_Noise` to something else and keep it as a second `cNoise` instance.